src/components/SearchForm/SearchForm.js

```diff
@@ -19,7 +21,7 @@ function SearchForm({ handleSearchClick }) {
         onChange={handleSearchChange}
         value={searchText || ''} >
       </input>
-      <button type="submit" className='search__button' onClick={handleSearchClick}>Search</button>
+      <button type="submit" className={`search__button ${searchButtonClassDisabled}`} onClick={handleSubmit}>Search</button>
```

Comment:
` onClick={handleSubmit}>`

please, delete it from the button

Submit handlers are added only to tags `form` with event `submit` rather than to submit buttons with event `click` because it also automatically handles `Enter` presses